source/docs/articles/sites/database/kill-mysql-queries.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Identify and Kill Queries with MySQL Command-Line Tool
+description: Learn how to identify and kill long-running MySQL queries on your WordPress or Drupal site in a few commands.
+keywords: mysql, mysql query, mysql queries, kill query, kill queries, kill mysql, kill mysql queries, queries, query, debug mysql, debug, mysql error
+---
+Long-running MySQL queries keep other transactions from accessing the necessary tables to execute a request, leaving your users on hold. To kill these queries, all you need is a local MySQL client such as [MySQL Workbench](http://dev.mysql.com/downloads/tools/workbench/), [Sequel Pro](http://www.sequelpro.com/download), or [Navicat](http://www.navicat.com/download).

```

Comment:
Suggest changing 2nd sentence to "To kill these queries, you'll need to [Access the appropriate environment's MySQL Database](/docs/articles/local/accessing-mysql-databases).